CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-## 1.23.0 - 2023-27-14
+## 1.23.X - 2023-*
+- Add setEntityCounter, getEntityCounter methods to Experiment
```

Comment:
Here you're referring to them as `setEntityCounter` and `getEntityCounter` but the function names are `setEntitySequence` and `getEntitySequence` .